Gateway/Http/Client/TransactionPayment.php

```diff
@@ -204,6 +230,8 @@ private function handleGiftcardPayments(
             $response = $service->payments(new PaymentRequest($giftcardPaymentRequest));
```

Comment:
Related to my first comment, what would you think about failing the order immediately here after getting an error or refusal here synchronously? IMO, it doesn't make sense to proceed to the next payments if the previous one already fails. Eventually, Adyen order will fail since one part has already failed.